### ISSUE TITLE
keyboard navigation(tab) dosnt work 

### ISSUE THREAD

**Issue by amitgilad** [JeremyEnglert/JointsWP-CSS#13]
for some reason i cant get the keyboard navigation to work, what am i missing
link:http://www.amitgilad.com/fnd

**Comment by amitgilad** [comment]
i forrced the offcanvas menu to be displyaed for small devices only and it is working now
